### ISSUE TITLE
site: add icons to landing page

### ISSUE THREAD

**Issue by jaketrent** [pluralsight/design-system#40]
icon assets to download: https://pluralsight.slack.com/files/dmitry-borodyansky/F6F5AUW6T/ds-bg.svg

**Comment by jaketrent** [comment]
Justing is whipping up some sweet illustrations.  Waiting on that.

**Comment by jaketrent** [comment]
I think is no longer relevant given the current robotic design
